src/main/webapp/WEB-INF/jsp/editCalendarUrl.jsp

```diff
@@ -27,7 +27,7 @@
 
     <h4><spring:message code="edit.calendar"/></h4>
 
-    <portlet:actionURL var="postUrl"><portlet:param name="action" value="editUrl"/></portlet:actionURL>
+    <portlet:actionURL var="postUrl" escapeXml="false"><portlet:param name="action" value="editUrl"/></portlet:actionURL>
```

Comment:
:+1: Needed to differentiate between characters used (i.e. &) and those that are already escaped